NoSQL_Schema

```diff
@@ -0,0 +1,54 @@
+
+db.addUser({
```

Comment:
How can you not install the db, and write queries without installing the db?
This is not the right way to insert data.